plugins/services/src/js/reducers/serviceForm/Volumes.js

```diff
@@ -21,6 +22,17 @@ const mapLocalVolumes = function (volume) {
   };
 };
 
+const mapExternalVolumes = function (volume) {
+  if (this.docker &&
+    this.dockerType === 'DOCKER' && volume.external.size != null) {
+    return Object.assign({},
```

Comment:
Is addressed I had to `omit` the `omit` but instead do the second Object.assign. because we need the `size: null`